docs/semgrep-assistant/overview.md

```diff
@@ -0,0 +1,116 @@
+
+Semgrep Assistant does not support the use of GitHub Enterprise Server (self-hosted).
+
+### Language support
+
+Semgrep Assistant supports [the same languages as Semgrep Code](/supported-languages).
+
+## Features
+
+### Component tags
+
+**Component tags** use GPT-4 to categorize a finding based on its function, such as:
+
+- Payments
+- User authentication
+- Infrastructure
+
+By categorizing your code through component tags, Semgrep Assistant can help you prioritize **high-risk issues**, such as remediating a code finding related to payments or user authentication.
+
+![Semgrep Assistant Component tag list](/img/assistant-component-tags.png)
+
+### Rules recommendations
+
+Semgrep Assistant suggests rule categories for your rules through the Assistant recommendations in the **Dashboard** page. To accept the suggestion, click the **Accept** button, and Semgrep Assistant automatically updates the rule with the suggested category.
+
+### Auto-triage
+
+Semgrep Assistant uses GPT-4's understanding of programming languages and libraries and your code and triage history to auto-triage findings and indicate if a finding can safely be ignored. For every recommendation to ignore a finding, Semgrep also provides guidance with an explanation on why this is the case.
+
+It can then surface its recommendations through PR or MR comments, so developers can fix or triage an issue without switching contexts. Semgrep Assistant can also send notifications through Slack.
+
+![Screenshot of Semgrep Assistant in a Slack notification](/img/semgrep-assistant-slack.png#md-width)
+    *Figure*. Semgrep Assistant in a Slack notification.
+
+You can view findings that Assistant suggests should be **Ignored** in Semgrep Cloud Platform's **Findings page**. <!-- #10438 -->
+
+### Autofix
+
+Semgrep Assistant can suggest [autofix](/writing-rules/autofix/) code snippets for Semgrep rules that don't have human-written autofix suggestions when it identifies a true positive. Assistant customizes the code snippets it provides based on previous feedback, if any, and your rule customizations. For example, if you have a custom rule that recommends a specific sanitizer, then Assistant will recommend its use in the autofix suggestion.
+
+ ou can set the minimum autofix confidence level required to display autofix suggestions from Semgrep Assistant.
+
+This feature is only active in PR and MR comments. Developers can review and verify Semgrep's generated fixes before they're applied.
+
+![ Screenshot of Semgrep Assistant generating a potential fix](/img/semgrep-assistant-autofix.png)
+*Figure*. Semgrep Assistant generates a potential fix.
+
+You can view findings for which Assistant has generated autofix snippets in Semgrep Cloud Platform's **Findings page**. <!-- #10438 --> The finding's detail page also includes a link to the PR or MR comment with the autofix since the PR or MR allows for directly committing the autofix. <!-- #10516 -->
```

Comment:
How do I view this?

Reply:
To clarify: there's no steps to view it, and the screenshot is that of the 'Assistant thinks this is a False Positive' type of feature (TP/FP assessment).

Reply:
I updated screenshot and reworded a bit to clarify (hopefully)